src/components/Breadcrumbs/Breadcrumbs.scss

```diff
@@ -0,0 +1,16 @@
+.breadcrumb {
+  margin-top: 6vh;
+
+  &__item {
+    margin: 10px;
+    
+    &--selected {
+      color: $primary-1;
+    }
+  }
+
+  &__item:hover {
+    color: $primary-1;
+  }
+  
+}
```

Comment:
Github likes empty lines at the bottom of files. Could you config your editor to do this?

<img width="305" alt="Screen Shot 2021-01-25 at 1 37 07 PM" src="https://user-images.githubusercontent.com/13306885/105769344-8a57ec00-5f12-11eb-9a39-7b3a1d2e9910.png">


```suggestion
}

```